srcpkgs/calicoctl34/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'calicoctl34'
+pkgname=calicoctl34
+_pkgname=calicoctl
```

Comment:
why even define it if you're going to use only once ?

Just remove it and put wrksrc="calico-${version}"